lib/src/dbs/executor.rs

```diff
@@ -54,11 +54,17 @@ impl<'a> Executor<'a> {
 		}
 	}
 
+	/// Commits theh transaction if it is local.
+	/// This function takes two additional parameters that are not present in the underlyingn kvs::Transaction::commit() function:
+	/// ns: The namespace of the transaction
+	/// db: The database of the transaction
+	/// These parameters are used updating the changefeed.
+	///
```

Comment:
I believe this is the "biggest" change to the existing codebase in this PR although the LOC change isn't actually large.

Presuming we do a commit per transaction that is tied to a namespace and a database, we take those two as the parameters, so that we can transactional update the change feed(which is, by its nature, tied to a namespace and a database)

Reply:
@phughk It turned out we (still) support cross namepace and cross database transactions 🤦
Honestly speaking, I don't know if it's practical in the long term.
Anyway, I reverted the changes to the commit function's signature and instead modified the change feed recorder to buffer changes along with the namespace and the database used when the change is made.
That way, although you don't have a single change feeds that spans namespaces and databases, you still have multiple database change feeds that can be concurrently updated via a single transaction.